test/__snapshots__/import-option.test.js.snap

```diff
@@ -755,36 +625,50 @@ Array [
   Array [
     "./import/import.css",
     "@import url(http://example.com/style.css);",
+    "",
+    true,
   ],
   Array [
     "./import/import.css",
     "@import url(http://example.com/style.css);",
+    "",
+    true,
```

Comment:
Do they need to be de-duplicated? It's the `url` case.
If the de-duplication is necessary in all cases, probably we should change the runtime https://github.com/webpack-contrib/css-loader/pull/1040#discussion_r369365652.